lib/apiservers/engine/backends/backends.go

```diff
@@ -212,8 +196,18 @@ func ProductVersion() string {
 	return productVersion
 }
 
+// VchConfig gets the VIC appliance VM's configspec
 func VchConfig() *config.VirtualContainerHostConfigSpec {
-	return vchConfig
+	var vchConfig config.VirtualContainerHostConfigSpec
```

Comment:
VIC Product wants to be able to dynamically load configuration.  We're being forced to do this.